testfiles/gedcom70/date-all.ged

```diff
@@ -0,0 +1,10337 @@
+﻿0 HEAD
+1 GEDC
+2 VERS 7.0
+0 @SIMPLE@ INDI
+1 BIRT
+2 NOTE 1401
+2 DATE 1401
+1 BIRT
+2 NOTE 1401/8
```

Comment:
The NOTEs provide the corresponding 5.5.1 date linevalues, but seem redundant and a bit strange when in a 7.0 sample file.